### PR TITLE
Translate to Traditional Chinese.

### PR DESCRIPTION
**Pull request by Hsins**
## Description of the Change

Translate to Traditional Chinese.

## Applicable Issues

issue #66

### CODE REVIEW

File: public/_locales/zh_TW/messages.json [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+					"sent": "每週工作日中，每小時寄送的郵件數"
+				}
+			},
+			"leader": {
+				"title": {
+					"received": "收到最多郵件來自",
+					"sent": "寄送最多郵件去往"
+				},
+				"description": {
+					"received": "每位收件人發送的電子郵件數量”,
```
Comment (devmount) [inline public/_locales/zh_TW/messages.json:60]: There is a wrong quotation character at the end of this line

**Review by devmount — CHANGES_REQUESTED** [review]
Many thanks for this contribution. Somehow there is a wrong quotation character at the end of line 60, please fix this to make the json file valid again and I will approve and merge this PR.

**Review by devmount — APPROVED** [review]
Looks awesome now, thanks again for your contribution!

File: public/_locales/zh_TW/messages.json (hunk 1 above)
Comment (Hsins) [reply public/_locales/zh_TW/messages.json]: Yes, I found the wrong quotation character `”` and fix it already.

**Pull request merged**